Remotion/Web/Development.WebTesting.IntegrationTests/FileDownloadTest.cs

```diff
@@ -475,20 +475,10 @@ private void CleanupPartialSampleFiles (string downloadDirectory, IEnumerable<st
 
     private Task ReinitializeFixtureAfterDelayAsync (TimeSpan delay)
     {
-      return Task.Run (
-          () =>
-          {
-            try
-            {
-              Task.Delay (delay)
-                  .ContinueWith (_ => IntegrationTestTearDown())
-                  .ContinueWith (_ => IntegrationTestTestFixtureTearDown());
-            }
-            finally
-            {
-              IntegrationTestTestFixtureSetUp();
-            }
-          });
+      return Task.Delay (delay)
+              .ContinueWith (_ => IntegrationTestTearDown())

```

Comment:
hmm...am thinking this could be written with method references instead of anonymous methods

Reply:
You mean `.ContinueWith (IntegrationTestTearDown)`? That's not possible because `ContinueWith` expects a method that accepts a Task as its parameter, hence the discard

Reply:
*blinks* Do'oh!